windows/mpconfigport.h

```diff
@@ -45,6 +45,8 @@
 #define MICROPY_DEBUG_PRINTERS      (1)
 #define MICROPY_USE_READLINE_HISTORY (1)
 #define MICROPY_HELPER_REPL         (1)
+#define MICROPY_REPL_EMACS_KEYS     (1)
```

Comment:
Hm, strange, https://github.com/micropython/micropython/commit/7d588b0c7c9698d23b1e5e7b54e0e2094ad8f464 says "Disabled by default", but actually enables it. I guess that's what needs fixing. @pramasoul: FYI.